package.json

```diff
@@ -23,7 +23,7 @@
     "framer-motion": "^6.2.8",
     "idb-keyval": "^6.1.0",
     "localforage": "^1.10.0",
-    "many-js": "l-1-labs/many-js#75402779abd8f468de95cc910e9be9454a8f573a",
+    "many-js": "tantommy/many-js#32bfcff4e4e8ba3594ad640f7a3ea82ee932dca6",
```

Comment:
Don't forget to update this

Reply:
atm, it is up to date with https://github.com/tantommy/many-js/commits/tt/webauthn PR, but i'll change it to the main branch from upstream whenever PR gets merged

Reply:
@hansl , has been updated and build output can be found here https://github.com/tantommy/albert/actions/runs/2386056231